src/dmd/escape.d

```diff
@@ -1292,3 +1292,26 @@ void findAllOuterAccessedVariables(FuncDeclaration fd, VarDeclarations* vars)
         }
     }
 }
+
+/***********************************
+ * Turn off STC.maybescope for variable `v`.
+ * This exists in order to find where STC.maybescope is getting turned off.
+ * Params:
+ *      v = variable
+ */
+version (none)
+{
+    void notMaybeScope(string file = __FILE__, int line = __LINE__)(VarDeclaration v)
+    {
+        printf("%.*s(%d): notMaybeScope('%s')\n", cast(int)file.length, file.ptr, line, v.toChars());
+        v.storage_class &= ~STC.maybescope;
+    }
+}
```

Comment:
How about defining a special `opAssign` on STC which might be a more generic debugging facility?

Reply:
Because I don't want a blizzard of messages to sort through, and because I've had repeated issues only with maybescope, not others.